src/web/pages/overrides/detailspage.jsx

```diff
@@ -109,9 +108,11 @@ ToolBarIcons.propTypes = {
   onOverrideEditClick: PropTypes.func.isRequired,
 };
 
-const Details = connect(rootState => ({
-  timezone: getTimezone(rootState),
-}))(({entity, timezone, ...props}) => {
+const Details = connect(rootState => {
+  return {
```

Comment:
Is this change to explicit return intentional ?

Reply:
No, it's not needed. Reverted.